girder/utility/webroot.py

```diff
@@ -57,6 +56,25 @@ def updateHtmlVars(self, vars):
         self.vars.update(vars)
         self.indexHtml = None
 
+    def addTemplateDirectory(self, templateDir):
+        """
+        Add a directory in which to search for templates.
+        """
+        self._templateLookup.directories.append(templateDir)
+
+    @property
+    def templateFilename(self):
+        """
+        The filename of the template to render. The file should be located in a
+        directory specified by addTemplateDirectory().
+        """
+        return self._templateFilename
```

Comment:
If `self.indexHtml` is gone (which I think it should be), does the getter / setter do anything? Could you just make `self._templateFilename` -> `self.templateFilename` and eliminate the methods?